src/App.tsx

```diff
@@ -26,70 +27,139 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  switch (sortType) {
+    case 1:
+      visibleGoods.sort((g1, g2) => {
+        return g1.localeCompare(g2);
+      });
+
+      break;
+
+    case 2:
```

Comment:
Instead of using numbers as case values in the switch statement, use the SortType enum for better readability and maintainability